pandas/core/resample.py

```diff
@@ -860,6 +860,25 @@ def size(self):
             from pandas import Series
 
             result = Series([], index=result.index, dtype="int64")
+        elif not len(self.ax) and isinstance(self._selected_obj, ABCSeries):
```

Comment:
Can `self._selected_obj` be anything but a Frame or Series? If not can probably just blow away this condition altogether